cmd/util/cmd/read-badger/cmd/protocol_kvstore.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/state/protocol/protocol_state/kvstore"
+
+	"github.com/onflow/flow-go/cmd/util/cmd/common"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+var flagDecodeData bool
+
+func init() {
+	rootCmd.AddCommand(protocolStateKVStore)
```

Comment:
We should add to write output to a file.

Reply:
I find the ability to redirect the output works well enough. We can do `util read-badger protocol-kvstore --block-id 1234 > $FILE` to write to $FILE.